capstone/src/main/webapp/dashboard-loader.js

```diff
@@ -31,13 +31,21 @@ function getClassrooms() {
     });
 }
 
+// TODO: Replace dummy values with real values in future PR
 function createClassroomDivElement(classroom) {
     let domparser = new DOMParser();
     let doc = domparser.parseFromString(`
-            <div class="classroom">
-                <h2>${classroom.entity.propertyMap.subject}</h2>
-                <p><a href="form.html">Classroom Link</a></p>
+            <div class="card margin margin-left">
+                <img class="card-img-top" src="/assets/soundwave.svg" alt="Room Card">
+                <div class="card-body text-center">
+                    <h5 class="card-title">${classroom.entity.propertyMap.subject}</h5>
```

Comment:
This could be a code injection issue, as we talked about in standup. Feel free to ping me if you want to talk more on this.

Reply:
Will do! I think we're planning on handling this in a future PR